tests/test_cli/test_cli.py

```diff
@@ -15,6 +15,7 @@ def capture(command,instream=None):
 
 
 def test_help(python_version):
+    os.chdir(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
```

Comment:
**Nitpick** : Consider using something else if you want to go up three directories. Use `os.pardir`

Reply:
@huguesdevimeux Done :) I have made it change back to the original directory in `test_cfg_subcmd` itself, so there is no reason to do it here,